.github/workflows/ci.yaml

```diff
@@ -77,3 +67,15 @@ jobs:
 
       - name: Check for unstaged files
         run: git diff --exit-code
+  fmt:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - uses: actions/setup-go@v5
+        with:
+          go-version: "~1.21"
+
+      - name: Check format
+        run: bash ./scripts/check_fmt.sh
```

Comment:
It's better to not call `bash` here and execute the script directly (make sure script is `chmod +x`). Otherwise I believe this will override the `#!` header in the script file meaning it _could_ run in an incompatible shell.

Reply:
Good call, I already had set the permissions with `chmod +x`